<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>accountWizard</class>
 <widget class="QWizard" name="accountWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Account Wizard</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/images/bitshares_logo.png</normaloff>:/images/images/bitshares_logo.png</iconset>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ModernStyle</enum>
  </property>
  <widget class="QWizardPage" name="introPage">
   <widget class="QStackedWidget" name="introStack">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>571</width>
      <height>271</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="basicPage">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>551</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Options:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QRadioButton" name="rNewBrain">
         <property name="text">
          <string>Create new account (brain key)</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rNewPass">
         <property name="text">
          <string>Create new web-wallet account (password)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rOldBrain">
         <property name="text">
          <string>Import existing account (brain key)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rOldPass">
         <property name="text">
          <string>Import existing web-wallet account (password)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rOldKeys">
         <property name="text">
          <string>Import existing account (private keys)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="sweepPage">
     <widget class="QGroupBox" name="groupBoxR">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>551</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Options:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QRadioButton" name="rRepBrain">
         <property name="text">
          <string>Generate new private keys (brain key)</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rRepPass">
         <property name="text">
          <string>Generate new private keys (web-wallet password)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rRepKeys">
         <property name="text">
          <string>Use pre-generated private keys</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QWizardPage" name="passwordPage1">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>521</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="horizontalSpacing">
      <number>20</number>
     </property>
     <property name="verticalSpacing">
      <number>20</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Account Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="inventAccount"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="inventPassword">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="generatePassword">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Unless you also export your private keys, losing this password will yield complete account lock-out and loss of funds.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>-50</x>
      <y>70</y>
      <width>193</width>
      <height>256</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/images/images/old/padlock.png</pixmap>
    </property>
   </widget>
   <zorder>label_16</zorder>
   <zorder>formLayoutWidget</zorder>
  </widget>
  <widget class="QWizardPage" name="passwordPage2">
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="horizontalSpacing">
      <number>20</number>
     </property>
     <property name="verticalSpacing">
      <number>20</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Account Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="oldAccount"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="oldPassword"/>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="passwordConfirm">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>80</y>
      <width>193</width>
      <height>256</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/images/images/old/padlock.png</pixmap>
    </property>
   </widget>
   <zorder>label_15</zorder>
   <zorder>formLayoutWidget_2</zorder>
  </widget>
  <widget class="QWizardPage" name="brainPage1">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>-60</y>
      <width>211</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/images/images/old/images.png</pixmap>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="brainkeyView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>521</width>
      <height>161</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>You must remember (or write down) this Brain Key - it could be used to restore your account at any later date, using any BitShares software.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="brainPage2">
   <widget class="QBrainKeyEdit" name="brainkeyEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>521</width>
      <height>161</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhLatinOnly|Qt::ImhMultiLine|Qt::ImhUppercaseOnly</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>-60</y>
      <width>211</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/images/images/old/images.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter your Brain Key:</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>brainkeyEdit</zorder>
   <zorder>label_4</zorder>
  </widget>
  <widget class="QWizardPage" name="registerPage">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_4">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="horizontalSpacing">
      <number>20</number>
     </property>
     <property name="verticalSpacing">
      <number>20</number>
     </property>
     <item row="0" column="1">
      <widget class="QComboBox" name="faucetBox"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="registrarBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Account Name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="accountName"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>Owner Key</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="pubkeyOwner"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Active Key</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="pubkeyActive"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Memo Key</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="pubkeyMemo"/>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="faucetComment">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="faucetRadio">
       <property name="text">
        <string>Faucet</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="registrarRadio">
       <property name="text">
        <string>Registrar</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWizardPage" name="keysPage">
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <property name="horizontalSpacing">
      <number>20</number>
     </property>
     <property name="verticalSpacing">
      <number>20</number>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Private Keys</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPlainTextEdit" name="privateKeys">
       <property name="font">
        <font>
         <family>Monospace</family>
        </font>
       </property>
       <property name="plainText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Account Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="accountEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="accountStatus">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="privkeysStatus">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QBrainKeyEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>bitsharesqt.qbrainkeyedit</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
